### PR TITLE
bust cache

### PR DESCRIPTION
**Pull request by rasummer**
@octonato hopefully this busts the css and js cache. Also I included 'Built by Lightbend' in this PR

### CODE REVIEW

**Review by octonato — APPROVED** [review]
LGTM

File: index.html [hunk 1]
```diff
@@ -148,7 +148,7 @@ <h2>Reactive Streaming Data</h2>
 
 <section class="wrapper sponsors">
     <div class="row">
-        <h1>Sponsored by</h1>
+        <h1>Built by</h1>
```
Comment (raboof) [inline index.html:151]: Do we really want to say "Built by Lightbend"? I agree "Sponsored" is probably too weak - Lightbend does a lot more than just sponsoring - but we always have a healthy amount of outside contributions as well...

Comment (octonato) [reply index.html:151]: oh sorry @raboof, I went too fast

Comment (octonato) [reply index.html:151]: Any suggestion on how to phase it?

**Pull request merged**